ArdentHQ/dashbrd
# refactor: use scaled endpoint for nft images

## Summary
Closes https://app.clickup.com/t/861nbbrf3

Closes https://github.com/ArdentHQ/dashbrd/issues/66

## For testing:

#### 1. Testing with new images (nft collection update)
* Visit an nft page in the app, take the token number, 
* Note that the nft image should have `thumbnailv2` in it's path,
* Replace `$this->startToken` with the token number copied above (so that it will update the viewing nft first) https://github.com/ArdentHQ/dashbrd/blob/baae8dd5613ccc5acdf0820ba597289fa9d64ffd/app/Jobs/FetchCollectionNfts.php#L53 
* Run horizon and then `php artisan collections:fetch-nfts --collection-id=18` (make sure to provide the correct collection id that the nft belongs to)
* After the first run of `FetchCollectionsNfts` job, refresh the page and check the nft image url. It should have `scaled` in its path, instead of `thumbnailv2`

#### 2. Testing existing nft images (migration)
* Checkout to `develop`, and run `composer db:live`
* Check one nft in db to see that their `large,small,thumb`  images in `extra_attributes` have `w_512,h_512/thumbnailv2`, `w_256,h_256/thumbnailv2`, and `w_96,h_96/thumbnailv2`  respectively in their paths 
* Now checkout to `refactor/scaled-nft-images` and run again `composer db:live`
* Check the same nft to see that the above paths have changed to `w_512/scaled`, `w_256/scaled`, `w_96/scaled`.







## Checklist
